pynance/test_data/dkb_visa_sample.csv

```diff
@@ -1,7 +1,7 @@
 "Kreditkarte:";"3546********6546";
 
 "Zeitraum:";"letzten 60 Tage";
-"Saldo:";"465.33 EUR";
+"Saldo:";"465,33 EUR";
```

Comment:
Did DKB provide a `,` in the first place? Or do they provide a `.`? Anyways, I'd vote to use the same separator DKB exports if this is not already the case.

Reply:
DKB uses commas (german locale), but when I put in fake data - out of habit, I used the dot. Found this error when I ran the tests...